Update time.py (for real this time)

LegitSi opened the pull request:
sorry but i had several skill issues, this SHOULD be fixed now

LegitSi committed:
Update time.py

sorry but i had several skill issues, this SHOULD be fixed now

infernostars commented on cogs/time.py:
```diff
@@ -27,12 +27,12 @@ async def time(self, interaction: discord.Interaction):
-      except e as Exception:
-          error_embed = error_template(f"Conversion to readable time failed. Fuck.\n{logging.log.error(e)}")
+      except Exception as e:
+          error_embed = error_template(f"Time command failed.\n{logging.log.error(e)}")
```
the logging.log stuff will log it out to the console, so seperate it off
also logging.log.exception() on its own will work [not your fault, me being dumb]

LegitSi replied:
real

LegitSi committed:
Update time.py

i dont know what you mean by separate it off but uhhh Real fixed the other Issue

infernostars commented:
why are you putting it in the fstring still, it doesnt return anything, it just logs it
it should be a seperate line

LegitSi committed:
Update time.py

okay this SHOULD be fixed i think i think fuck

infernostars commented:
its not perfect but /shrug
